.travis.yml

```diff
@@ -6,16 +6,21 @@ git:
   # don't limit git clone depth
   depth: false
 
+matrix:
+  include:
+    - os: linux
+    - os: linux-ppc64le
+
 before_install:
   - docker pull golang:1.11.1
-  - docker pull node:6.11.0-slim
+  - docker pull node:8.15.0-slim
   - docker pull f5devcentral/containthedocs
 
 script:
   - set -e
-  - docker run --rm -it -v `pwd`:`pwd` -w `pwd` node:6.11.0-slim build-tools/schema-tests.sh
+  - docker run --rm -it -v `pwd`:`pwd` -w `pwd` node:8.15.0-slim build-tools/schema-tests.sh
   - if [ "$DOCKER_NAMESPACE" == "" ]; then DOCKER_NAMESPACE="local"; fi
-  - BASE_PUSH_TARGET="$DOCKER_NAMESPACE/k8s-bigip-ctlr"
+  - BASE_PUSH_TARGET="$DOCKER_NAMESPACE/k8s-bigip-ctlr-`uname -m`"
```

Comment:
I prefer pushing images like the following:

`F5Networks/k8s-bigip-ctlr:<tag>-<ARCH>` e.g `F5Networks/k8s-bigip-ctlr:<tag>-amd64`,`F5Networks/k8s-bigip-ctlr:<tag>-ppc64le` and push `F5Networks/k8s-bigip-ctlr:<tag>` as fat manifest so that all the remaining manifest files are intact.